README.md

```diff
@@ -474,154 +62,18 @@ DF_INSTITUTION_NAME="University of Foo"
 
 You can also keep multiple `.env` files for different environments, e.g.: `.env.production` is different to `.env.develoment`. Doubtfire uses the [dotenv](https://github.com/bkeepers/dotenv) gem to make this happen.
 
-##### Database Population
-
-Whilst still in the Doubtfire API project root, execute:
-
-```
-$ rake db:create
-```
-
-You can choose to populate the database with some fake test data using:
-
-```
-$ rake db:populate
-```
-
-#### 7. Get it up and running!
 
+#### 3. Get it up and running!
+Once you've installed using either in install script or the manual install steps,
 Run the Rails server and check the API is up by viewing Grape Swagger documentation:
 
 ```
 $ rails s
+$ open http://localhost:3000/api/docs/
```

Comment:
Open will only work for mac, not Linux. This line isn't needed as we explain it on the following line.